### PR TITLE
refactor sample launches in jsk_perception

### PR DESCRIPTION
**Pull request by knorth55**
- remove `test_mode` in `sample_ssd_object_detector.launch`
- remove `test_mode` in `sample_face_pose_estimation.launch`
- use `gpu` instead of `GPU` in sample launches

### CODE REVIEW

**Comment by knorth55** [comment]
`test_mode` is complicated to use with own custom args.

**Comment by knorth55** [comment]
Small letters: Users can change it on command line, like `roslaunch ~~~~.launch gpu:=0`
Capital letters: users are not recommended to change by command line. Users should change their values in launch files with `<include>` tag

File: jsk_perception/launch/face_pose_estimation.launch [hunk 1]
```diff
@@ -1,12 +1,8 @@
 <launch>
-  <arg name="test_mode" default="false" />
-  <arg name="gui" default="true" unless="$(arg test_mode)" />
-  <arg name="gui" default="false"    if="$(arg test_mode)" />
-  <arg name="GPU" default="0" unless="$(arg test_mode)"  />
-  <arg name="GPU" default="-1"    if="$(arg test_mode)" />
+  <arg name="gui" default="true" />
+  <arg name="GPU" default="-1" />
```
[the remark this answers is not in the document]
Comment (knorth55) [reply jsk_perception/launch/face_pose_estimation.launch]: OK, I will change,
BTW, this PR follows rules below. Do you agree with this rule?
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2376#issuecomment-446883632

[the remark this answers is not in the document]
Comment (iory) [reply jsk_perception/launch/face_pose_estimation.launch]: I agree with it. 
Also we should write this rule to somewhere

[the remark this answers is not in the document]
Comment (knorth55) [reply jsk_perception/launch/face_pose_estimation.launch]: I add a contribution guideline page in doc.
contributions

**Pull request merged**